### ISSUE TITLE
TypeError: argument 1 must be string, not pyewf.handle

### ISSUE THREAD

**Issue by dmbina** [dlcowen/dfirwizard#1]
@dlcowen When I run dfirwizard-v5.py code to extract files from my E01 image. I got this message error:
 
![dfirwizard_typeerror](https://cloud.githubusercontent.com/assets/23559224/20439647/27e3c01c-ad82-11e6-8189-1b16de1ebdb2.PNG)

What should I do ? I mean this code makes perfect sense unless pytsk3.Img_Info is a string and the function ewf_Img_Info cannot take a pyewf.handle parameter ?

**Comment by dlcowen** [comment]
Looks like you have a typo, it's ewf handle not ef

Get Outlook for Androidhttps://aka.ms/ghei36

From: Delvain
Sent: Friday, November 18, 9:33 AM
Subject: [dlcowen/dfirwizard] TypeError: argument 1 must be string, not pyewf.handle (#1)
To: dlcowen/dfirwizard
Cc: David Cowen, Mention

@https://github.com/dlcowendlcowenhttps://github.com/dlcowen When I run dfirwizard-v5.py code to extract files from my E01 image. I got this message error:

What should I do ? I mean this code makes perfect sense unless pytsk3.Img_Info is a string and the function ewf_Img_Info cannot take a pyewf.handle parameter ?

##

**Comment by dmbina** [comment]
I did change the type from ef to ewf_handle but I still have the same error...
![dfirwizard_typeerror2](https://cloud.githubusercontent.com/assets/23559224/20450123/903b9f6e-adb3-11e6-80a7-c5a386bc1eea.PNG)

**Comment by dmbina** [comment]
You were right it was a typo issue. I solved it when I changed _init_ to __init__  . The double underscores in init got formatted as a single one. But I ran into another issue : 
![tsk3_error](https://cloud.githubusercontent.com/assets/23559224/20467620/b7cef3c0-af50-11e6-885f-bc82b79e1ee5.PNG)

**Comment by dlcowen** [comment]
Is it a full disk image or just a partition

Get Outlook for Androidhttps://aka.ms/ghei36

**Comment by dmbina** [comment]
It is a full disk image. I created an image of an USB drive (1G) containing two files using Pro Discover. 
Do i have to install SleuthKit so that pytsk3 can work properly ?

**Comment by dlcowen** [comment]
No, can you tell me what the partition table shows through the OS?

Get Outlook for Androidhttps://aka.ms/ghei36

**Comment by dmbina** [comment]
It shows this: 
![partition](https://cloud.githubusercontent.com/assets/23559224/20473883/28f25eec-af89-11e6-85b9-36e4db92bdd7.PNG)

**Comment by dlcowen** [comment]
Can you download and run mmls? It could be the whole disk was formatted and there is no partition table

Get Outlook for Androidhttps://aka.ms/ghei36

**Comment by dmbina** [comment]
I know how to run mmls on linux but I don't on windows. And I am actually on Windows 8.
But I got this by using diskpart

![partition_table](https://cloud.githubusercontent.com/assets/23559224/20515267/6c2f4440-b056-11e6-821a-3b243b2cc02d.PNG)

**Comment by dmbina** [comment]
I managed to get the Volume_Info code working:
![pytsk3_volume_info](https://cloud.githubusercontent.com/assets/23559224/20995293/a24ccd56-bcbb-11e6-85fc-8d2be4c38769.PNG)
 it replicates the mmls functionnality
But I still got stuck when I tried to access the file system:
![pytsk3_fs_info](https://cloud.githubusercontent.com/assets/23559224/20995377/3165611a-bcbc-11e6-82f5-6e800d384a7f.PNG)
